content/articles/account-collaborator.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: Differences between Account Collaborator and Account Membership
+excerpt: The differences between an Account Collaborator and an Account Member.
+categories:
+- Account
+---
+
+# Account Collaborators and Account Memberships
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+This article describes both the *Account Collaborator* feature and the *Account Membership* feature, and the differences between each.
+
+
+## Account Collaborator
+
+<warning>
+Account Collaborators are deprecated, and it is no longer possible to add an Account Collaborator. Instead, please use Account Membership when adding another user to your account for domain management.
+</warning>
+
+Previously, account access could be granted using the Collaborator feature. The Collaborator feature only provides access to DNS management. It does not allow domain management, account management or any other purchase action.
+
+
+## Account Membership
+
+<info>
+To have multiple users on one account, you must be [subscribed to an eligible plan](https://dnsimple.com/pricing).
+</info>
+
+When you add a user to an account, they will have full access rights to every resource attached to the account, including domains, contacts, SSL certificates, etc.
+
+All members will also have full-admin access to the account, including the ability to access and change billing information and account details. The user will retain their login credentials and two-factor authentication configuration.
+
+You can find out more information about how to add an Account Member [here](https://support.dnsimple.com/articles/account-users/).
```

Comment:
I moved the link in the opening, to settle the context. FYI please use relative links when cross-linking articles.

Reply:
Thanks Simone - I'll do that in future!